torch/csrc/jit/init.cpp

```diff
@@ -119,17 +113,25 @@ void initJITBindings(PyObject *module) {
           py::arg("optimize") = true)
       .def(
           py::init([](std::shared_ptr<Graph> graph, bool optimize) {
-            return createExecutorByGraph(std::move(graph), optimize);
+            return GraphExecutor(std::move(graph), optimize);
           }),
           py::arg("graph"),
           py::arg("optimize") = true)
-      .def("__call__", [](GraphExecutor& ge, py::args args) {
+      .def("__call__", [](GraphExecutor& ge, py::args args) -> py::object {
         auto inputs = createVariableTensorList(args);
         auto outputs = ge.run(std::move(inputs));
         // if we don't tell pybind these are variables it chokes on the
         // conversion.
         // TODO: fix conversions to be sane and make sure this works.
```

Comment:
Is this comment still relevant?

Reply:
kind of -- we have to do the static_cast to Variable otherwise pybind tries to convert the tensors, and at::Tensor conversion doesn't work if the tensor is a Variable. I'd like to be able to drop the static_cast's but we are waiting for tensor/variable to merge to make it happen.